cellprofiler/gui/help.py

```diff
@@ -2027,287 +2027,323 @@
-each file represents a different stack.</li>
-<li>For each stack, distinguish the frames from each other. This information is usually contained
-in the image's internal metadata, in contrast to the image sequence described above.</li>
+<p>Another common means of storing time-lapse or Z-stack data is as a single file containing frames. Examples of this
+approach include image formats such as:</p>
+<ul>
+    <li>Multi-frame TIF</li>
+    <li>Metamorph stack: STK</li>
+    <li>Evotec/PerkinElmer Opera Flex</li>
+    <li>Zeiss ZVI, LSM</li>
+    <li>Standard movie formats: AVI, Quicktime MOV, etc</li>
+</ul>CellProfiler uses the Bio-Formats library for reading various image formats. For more details on supported files,
+see this <a href="http://www.openmicroscopy.org/site/support/bio-formats4/supported-formats.html">webpage</a>. In
+general, we recommend saving stacks and movies in .TIF format.
+<p><i>Example:</i> You have several image stacks in the following format:</p>
+<ul>
+    <li>The stacks are saved in .TIF format.</li>
+    <li>Each stack is a single-channel grayscale image.</li>
+    <li>Your files have names like IMG01_CH01.TIF, IMG01_CH02.TIF, ... IMG01_CH04.TIF and IMG02_CH01.TIF,
+    IMG02_CH02.TIF, etc, where IMG01_CH01.TIF designates channel 1 from image 1, IMG01_CH02.TIF designates channel 2
+    from image 1, and IMG02_CH01.TIF designates channel 1 from image 2.</li>
 </ul>
-To accomplish this, do the following:
-<ul>
-<li>Select "%(X_AUTOMATIC_EXTRACTION)s" as the metadata extraction method. In this case, CellProfiler will
-extract the requisite information from the metadata stored in the image headers.</li>
-<li>Click the "Update metadata" button. A progress bar will appear showing the time elapsed; depending on
-the number of files present, this step may take a while to complete.</li>
-<li>Click the "Update" button below the divider.</li>
-<li>The resulting table should show the various metadata contained in the file. In this case, the relevant
-information is contained in the <i>C</i> and <i>Series</i> columns. In the figure shown, the <i>C</i> column
-shows three unique values for the channels represented, numbered from 0 to 2. The <i>Series</i> column shows
-8 values for the slices collected in each stack, numbered from 0 to 7, followed by the slices for other stacks. </li>
+<p>In this case, the procedure to set up the input modules to handle these files is as follows:</p>
+<ul>
+    <li>In the <b>Images</b> module, drag-and-drop your folders of images into the File list panel. If necessary, set
+    your rules accordingly in order to filter out any files that are not images to be processed.<br>
+    In the above example, you would drag-and-drop the .TIF files into the File list panel.</li>
+    <li>In the <b>NamesAndTypes</b> module, select "Yes" for "Volumetric" to tell CellProfiler your images should be
+    processed as whole volumes. You can provide the voxel spacing in each dimension. CellProfiler will use this
```

Comment:
Definitely agree that "Volumetric" here would be opaque to most biologists. Alternate suggestions: "3D image stack" or "3D volume", if you want to avoid using question forms in the settings.